src/runtime/hexagon_remote/Makefile

```diff
@@ -78,6 +79,10 @@ remotes: bin/v60/libhalide_hexagon_remote_skel.so bin/v60/hexagon_sim_remote
 bin/src/halide_hexagon_remote.h bin/src/halide_hexagon_remote_skel.c bin/src/halide_hexagon_remote_stub.c: halide_hexagon_remote.idl
 	mkdir -p $(@D)
 	$(HEXAGON_QAIC) $(HEXAGON_QAICFLAGS) $^ -o $(@D)
+	mv -f bin/src/halide_hexagon_remote.h      bin/src/halide_hexagon_remote.h-orig 
```

Comment:
This is pretty messy... how about just making the param bwMegabytesPerSec instead of bwBytesPerSec, so it can just be a 32 bit int? And convert to bwBytesPerSec in the runtime?

You can even still keep the public API bwBytesPerSec if you wanted (HalideRuntimeHexagonHost.h), though that would be slightly misleading because some precision will be lost.